grpc.go

```diff
@@ -163,9 +160,11 @@ func (a *agentGRPC) onlineCPUMem(req *pb.OnlineCPUMemRequest) error {
 		return handleError(req.Wait, fmt.Errorf("requested number of CPUs '%d' must be greater than 0", req.NbCpus))
 	}
 
-	onlineCPUMemLock.Lock()
-	defer onlineCPUMemLock.Unlock()
+	// we are going to update the containers of the sandbox, we have to lock it
+	a.sandbox.Lock()
+	defer a.sandbox.Unlock()
 
+	agentLog.Debugf("Number of vCPUs to connect: %d", req.NbCpus)
```

Comment:
Nit: stylistically, I prefer to use `WithField()` or `WithFields()` for this message and others below:

```
agentLog.WithField("vcpus-to-connect", req.NbCpus).Debug("connecting vCPUs)
```

It also makes the logs easier+safer to parse as those fields are less likely to change.